src/styles/_fontcontent.scss

```diff
@@ -33,6 +33,16 @@
     .uk-select:not([multiple]):not([size]) {
     	height: $baseFontSize * 2.5;
     }
+
+    .uk-select,
+    .multiselect, .multiselect__input, .multiselect__single {
```

Comment:
Can you move this and the following style into _fontcontent.scss so that it it gets updated when the base font size changes per #115 ? Trying to keep font-size css isolated with that change.